src/server.app.src

```diff
@@ -4,10 +4,9 @@
   {registered, []},
   {mod, {server, []}},
   {applications,
-   [epgsql]},
-  {env,[]},
+   [kernel,stdlib,pure_migrations,epgsql]},
+  {env,[ {migrations_path, "/home/leto/Code/Personal/lyceum/database/migrations"} ]},
```

Comment:
This filepath seems to be too specific. Is that correct?

Reply:
I'm still not able to make this work, so I've tried hardcoding the path

Reply:
@EduardoLR10 reverted

Reply:
What errors are you facing? We can pair on this to debug it.

Reply:
I'll try the simplest, dumbest, idea first, which is instead of creating/modifying a module, just do an erlang script.